README.md

```diff
@@ -1,62 +1,63 @@
-#Notice
+# Notice
 
-Notice is a jQuery plugin for simple notifications
+Notice is a plugin for simple notifications.
 
-##Usage
+## Usage
 
-```javascript
-//info
-$.notice('message');
+- Import notice:
 
-//success
-$.notice('success!', { level: 'success' });
+```javascript
+  import Notice form 'notice';
 
-//error
-$.notice('error', { level: 'error' });
+  Notice('Test Notification');
 ```
 
-##Options
+- Import [default CSS styles](styles.css)
+
+## Notice(message, [options])
+
+`level` _{string='default'} Options: 'default', 'info', 'success', 'warning', 'error'}_ - Type of alert
```

Comment:
Can we move this into a table? (look at https://github.com/firstandthird/simple-domodal/tree/feature) You can use https://www.tablesgenerator.com/markdown_tables